website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -155,4 +155,38 @@ Row placement determines the visual styling based on where the row is placed rel
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
\ No newline at end of file
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Multi-selection
+
+A multi-select table includes checkboxes enabling users to select multiple rows in a table to perform bulk operations. Checking or unchecking the checkbox in the table header either selects or deselects the checkboxes on each row in the table body. Individual checkboxes in the rows can also be selected or deselected.
+
+Multi-selection consists of: 
+
+1. `Header Column / Select`: Used in the table's header row. This acts as the parent checkbox, allowing the selection or deselection of all child rows in a single table simultaneously.
+
+![Example of multi-selection in a table header](/assets/components/table/multi-select-header.png)
+
+2. `Cell / Select`: Used in each table row allowing for the selection of an individual row.
+
+![Example of multi-selection within table cells](/assets/components/table/multi-select-cells.png)
+
+
+### Intended interaction
+
+- When individual rows are selected, the parent checkbox in the Table header changes to display an indeterminate state. 
+- When no or some rows (but not all) are selected in a single Table, clicking the parent checkbox in the Table header will change to display as checked and all rows on that page will be selected.
+- When all rows are selected in a single Table, the parent checkbox in the header appears as checked. Clicking the parent checkbox will deselect all rows on that page.
+- An additional action outside of the Table is needed in order to select all rows across a paginated Table.
+
+!!! Info
+
+This documentation currently covers fundamental multi-select table interactions. We're working to include comprehensive patterns for additional actions. For questions or concerns, please reach out to [#team-design-systems](https://hashicorp.enterprise.slack.com/archives/C7KTUHNUS)
+!!!
```

Comment:
@hashibot-hds Following up on our discussion on adding a banner to convey future work, I added this info banner between the intended interaction section and an example to convey future pattern work and a place for consumers to reach out. I'm open to feedback on this; thank you!

Reply:
Maybe try this:

```suggestion
!!! Info

This documentation currently covers only fundamental multi-select table interactions. We're working to include more comprehensive patterns for handling actions when multiple rows are selected and how this impacts filtering and pagination within a data set. For questions or concerns, please reach out to [#team-design-systems](https://hashicorp.enterprise.slack.com/archives/C7KTUHNUS)
!!!
```